aibolit/patterns/send_null/send_null.py

```diff
@@ -9,59 +9,37 @@ class SendNull:
     def __init__(self):
         pass
 
-    def __find_position_recursively(self, node):
-        if not hasattr(node, 'children'):
-            return
-        else:
-            for i in node.children:
-                if not isinstance(i, list):
-                    if hasattr(i, '_position'):
-                        if i.position:
-                            return i._position.line
-                        else:
-                            for j in i.children:
-                                position = self.__find_position_recursively(j)
-                                if position:
-                                    return position
+    def __is_null(self, val):
+        has_ternary_true_value = hasattr(val, 'value')
+        if has_ternary_true_value:
+            is_ternary_true_str = isinstance(val.value, str)
+            if is_ternary_true_str:
+                if val.value == 'null':
+                    return True
```

Comment:
```suggestion
    def __is_null(self, val):
        if not hasattr(val, 'value'): 
            return False
        if not  isinstance(val.value, str): 
            return False
        if val.value != 'null': 
            return False
        return True
```

Reply:
this line will fail: 
`if not  isinstance(val.value, str): ` if val doesn't have value field. Your code will fail the program

Reply:
`if val doesn't have value field` I will exit with False before

Reply:
fixed